configure

```diff
@@ -121,6 +129,7 @@ parse_opts() {
             --dev)
                 WITH_DOCS=0
                 WITH_FAUXTON=0
+                WITH_NOUVEAU=1
```

Comment:
Are you sure this should default to on? It seems like maybe the least surprising thing would be to require explicitly enabling it with `--enable-nouveau`.